docs/codeql/codeql-overview/codeql-changelog/codeql-cli-2.20.4.rst

```diff
@@ -117,8 +117,8 @@ Java/Kotlin
 *   Deleted the deprecated :code:`isLValue` and :code:`isRValue` predicates from the :code:`VarAccess` class, use :code:`isVarWrite` and :code:`isVarRead` respectively instead.
 *   Deleted the deprecated :code:`getRhs` predicate from the :code:`VarWrite` class, use :code:`getASource` instead.
 *   Deleted the deprecated :code:`LValue` and :code:`RValue` classes, use :code:`VarWrite` and :code:`VarRead` respectively instead.
-*   Deleted a lot of deprecated classes ending in ``*Access``, use the corresponding ``*Call`` classes instead.
-*   Deleted a lot of deprecated predicates ending in ``*Access``, use the corresponding ``*Call`` predicates instead.
+*   Deleted a lot of deprecated classes ending in `*Access`, use the corresponding `*Call` classes instead.
+*   Deleted a lot of deprecated predicates ending in `*Access`, use the corresponding `*Call` predicates instead.
```

Comment:
```suggestion
*   Deleted a lot of deprecated classes ending in :code:`*Access`, use the corresponding :code:`*Call` classes instead.
*   Deleted a lot of deprecated predicates ending in :code:`*Access`, use the corresponding :code:`*Call` predicates instead.
```

Reply:
@jonjanego I think this bit broke the automation last time you did it as far as I can remember.  Here's an example of what `@felicitymay` did last time to fix them: https://github.com/github/codeql/pull/18718/files#diff-040b6d4c5e724e910141939b9fcd21f8f7bfc2ba7e4f7ca3ddf549bf90bdc66dR120